test/Infrastructure.IntegrationTest/Auth/Repositories/AuthRequestRepositoryTests.cs

```diff
@@ -72,6 +72,114 @@ public class AuthRequestRepositoryTests
+            // This last auth request is not created manually, and will be
+            // used to make sure entity framework's `UpdateRange` method
+            // doesn't create requests too.
+            CreateAuthRequest(user2.Id, AuthRequestType.AdminApproval, DateTime.UtcNow.AddMinutes(-11))
+        };
+
+        // Update some properties on two auth request, but leave the other one
+        // alone to be a control value
+        var authRequestToBeUpdated1 = authRequests[0];
+        var authRequestToBeUpdated2 = authRequests[1];
+        var authRequestNotToBeUpdated = authRequests[2];
+        authRequests[0].Approved = true;
+        authRequests[0].ResponseDate = DateTime.UtcNow.AddMinutes(-1);
+        authRequests[0].Key = "UPDATED_KEY_1";
+        authRequests[0].MasterPasswordHash = "UPDATED_MASTERPASSWORDHASH_1";
+
+        authRequests[1].Approved = false;
+        authRequests[1].ResponseDate = DateTime.UtcNow.AddMinutes(-2);
+
+        // Run the method being tested
+        await authRequestRepository.UpdateManyAsync(authRequests);
+
+        // Define what "Equality" really means in this context
+        // This includes stripping milliseconds off of dates, because we can't
+        // reliably compare that deep
+        static DateTime? TrimMilliseconds(DateTime? dt)
+        {
+            if (!dt.HasValue)
+            {
+                return null;
+            }
+            return new DateTime(dt.Value.Year, dt.Value.Month, dt.Value.Day, dt.Value.Hour, dt.Value.Minute, dt.Value.Second, 0, dt.Value.Kind);
+        }
+
+        bool AuthRequestEquals(AuthRequest x, AuthRequest y)
+        {
+            return
+            x.Id == y.Id &&
+            x.UserId == y.UserId &&
+            x.Type == y.Type &&
+            x.RequestDeviceIdentifier == y.RequestDeviceIdentifier &&
+            x.RequestDeviceType == y.RequestDeviceType &&
+            x.RequestIpAddress == y.RequestIpAddress &&
+            x.ResponseDeviceId == y.ResponseDeviceId &&
+            x.AccessCode == y.AccessCode &&
+            x.PublicKey == y.PublicKey &&
+            x.Key == y.Key &&
+            x.MasterPasswordHash == y.MasterPasswordHash &&
+            x.Approved == y.Approved &&
+            TrimMilliseconds(x.CreationDate) == TrimMilliseconds(y.CreationDate) &&
+            TrimMilliseconds(x.ResponseDate) == TrimMilliseconds(y.ResponseDate) &&
+            TrimMilliseconds(x.AuthenticationDate) == TrimMilliseconds(y.AuthenticationDate) &&
+            x.OrganizationId == y.OrganizationId;
+        }
+
+        // Assert that the unchanged auth request is still unchanged
+        var skippedAuthRequest = await authRequestRepository.GetByIdAsync(authRequestNotToBeUpdated.Id);
+        Assert.True(AuthRequestEquals(skippedAuthRequest, authRequestNotToBeUpdated));
+        //Assert.Equal(skippedAuthRequest.CreationDate, authRequestNotToBeUpdated.CreationDate);
```

Comment:
is this line meant to be commented?

Reply:
No, that's a leftover. Removed on https://github.com/bitwarden/server/pull/4053/commits/c6660d8de710d18a762a8fa57978f799b12639e7